CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-software-actions/compare/release-1.4.0...master
 
+### Fixed
+- Bump jira-actions to 3.18.0
```

Comment:
That's not that important from the perspective of the lib consumer. In the end, the consumer has the control over which transitive dependencies they use. They're only affected by major versions, which require more work.

```suggestion
```

Reply:
In this case it is importand, because we are bumping the minimum compatibility and it affects the consumer

Reply:
It doesn't affect the consumer, because they can bump minor version (automatically or not) without losing compatibility. We worked out that only requiring a new major version is breaking compat. That's already documented in CHANGELOG.md and our definition of API.

Reply:
right, indeed the lib compatibility restriction mentioned in the changelog may indicate a breaking change, however if we do have an exception from that case noted in changelog then it's not worth mentioning bumps there